src/System.Globalization/tests/System.Globalization.Tests.csproj

```diff
@@ -27,14 +27,7 @@
     <Compile Include="NumberFormatInfo\NumberFormatInfoPercentNegativePattern.cs" />
     <Compile Include="NumberFormatInfo\NumberFormatInfoPercentPositivePattern.cs" />
     <Compile Include="TextInfo.cs" />
-    <Compile Include="CompareInfo\compareinfo.cs" />
-    <Compile Include="CompareInfo\CompareInfoCompare.cs" />
-    <Compile Include="CompareInfo\CompareInfoCompare2.cs" />
-    <Compile Include="CompareInfo\CompareInfoIndexOf.cs" />
-    <Compile Include="CompareInfo\CompareInfoIndexOf2.cs" />
-    <Compile Include="CompareInfo\CompareInfoIsPrefix.cs" />
-    <Compile Include="CompareInfo\CompareInfoIsSuffix.cs" />
-    <Compile Include="CompareInfo\CompareInfoLastIndexOf.cs" />
+    <Compile Include="CompareInfo\CompareInfoTests.cs" />
```

Comment:
I don't think it would make sense for us to add other files in the future if we are going down the path of putting all the tests in 1 file now.